components/common/autoTeleport/AutoTeleportWelcomeModal.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <NeoModal
+    :value="isModalActive"
+    :can-cancel="['outside', 'escape']"
+    scroll="clip"
+    class="top"
+    @close="onClose">
+    <div class="modal-width">
+      <header
+        class="py-5 pl-6 pr-5 is-flex is-justify-content-space-between is-align-items-center">
+        <span class="is-capitalized is-size-3_5 has-text-weight-bold">
+          {{ $t('autoTeleport.welcomeToAutoTeleport') }}
+        </span>
+
+        <NeoButton
+          class="py-1 px-2"
+          variant="text"
+          no-shadow
+          icon="xmark"
+          size="medium"
+          @click="onClose" />
+      </header>
+
+      <div class="px-6 pt-4 pb-5">
+        <div class="welcome-card-container">
+          <div
+            v-for="(x, index) in 3"
+            :key="reasonsIcons[index]"
+            class="border border-k-shade p-4">
+            <div class="has-accent-blur">
+              <NeoIcon
+                :icon="reasonsIcons[index]"
+                size="large"
+                class="has-text-k-accent mb-3" />
+              <img src="/accent-blur.svg" class="blur icon-blur" alt="blur" />
+            </div>
+
+            <p class="has-text-weight-bold is-size-5 mb-3">
+              {{ $t(`autoTeleport.welcome.${x}.title`) }}
+            </p>
+
+            <p class="has-text-k-grey">
+              {{ $t(`autoTeleport.welcome.${x}.description`) }}
+            </p>
+          </div>
+        </div>
+
+        <p class="mt-4 is-capitalized">
+          {{ $t('autoTeleport.welcome.description') }}
+        </p>
+
+        <div class="mt-2">
+          <a
```

Comment:
link to?

Reply:
not available 
conversation with exezbcz 
![CleanShot 2023-11-07 at 09 41 15@2x](https://github.com/kodadot/nft-gallery/assets/44554284/879c2773-fc58-48c5-90de-e9e57bd389ea)

Reply:
alright if still not provided you should hide it (`.is-hidden`)